src/packages/src/xroad/common/proxy/usr/share/xroad/scripts/backup_xroad_proxy_configuration.sh

```diff
@@ -50,6 +50,18 @@ execute_backup () {
   fi
 }
 
+warn_about_localconf () {
+  if [ -f /etc/xroad/services/local.conf ]; then
+    echo
+    echo "=== Warning! ==========================================================================="
+    echo "The file “/etc/xroad/services/local.conf” used for configuration overrides is deprecated"
+    echo "and not included in the backups anymore."
+    echo "The file “/etc/xroad/services/local.properties” should be used instead."
+    echo "=== Warning! ==========================================================================="
+    echo
+  fi
+}
+
```

Comment:
When backing up the security server on the command line, the warning is now displayed twice; Once before and once after the backup operation. See the following terminal output.

```
xroad@ss1:/usr/share/xroad/scripts$ ./backup_xroad_proxy_configuration.sh  -s DEV/COM/1111/SS1 -f /var/lib/xroad/backup/mybackup.tar

The backup format has changed, and therefore, backups are incompatible between X-Road 6 and X-Road 7.

Also, please note that the "/etc/xroad/services/local.conf" file for configuration overrides is deprecated and is not included in the backups anymore. In version 7, the "/etc/xroad/services/local.properties" file should be used instead.

ENCRYPT_BACKUP=false
GPG_KEYIDS=
CREATING DATABASE DUMP TO /var/lib/xroad/dbdump.dat
CREATING SIGNED TAR ARCHIVE TO /var/lib/xroad/backup/mybackup.tar
security_XROAD_7.0_DEV/COM/1111/SS1
tar: Removing leading `/' from member names
/etc/xroad/
/etc/xroad/ssl/
/etc/xroad/ssl/internal.key
/etc/xroad/ssl/proxy-ui-api.p12
/etc/xroad/ssl/proxy-ui-api.crt
/etc/xroad/ssl/openssl.cnf.dpkg-dist
/etc/xroad/ssl/internal.crt
/etc/xroad/ssl/internal.p12
/etc/xroad/ssl/proxy-ui-api.key
/etc/xroad/ssl/openssl.cnf
/etc/xroad/globalconf/
/etc/xroad/globalconf/DEV/
/etc/xroad/globalconf/DEV/shared-params.xml.metadata
/etc/xroad/globalconf/DEV/shared-params.xml
/etc/xroad/globalconf/DEV/private-params.xml.metadata
/etc/xroad/globalconf/DEV/private-params.xml
/etc/xroad/globalconf/instance-identifier
/etc/xroad/globalconf/files
/etc/xroad/messagelog/
/etc/xroad/messagelog/messagelog.p12
/etc/xroad/backup.d/
/etc/xroad/backup.d/30_xroad-proxy
/etc/xroad/backup.d/10_xroad-signer
/etc/xroad/backup.d/20_xroad-monitor
/etc/xroad/backup.d/00_xroad-confclient
/etc/xroad/configuration-anchor.xml
/etc/xroad/autologin
/etc/xroad/jetty/
/etc/xroad/jetty/serverproxy.xml
/etc/xroad/jetty/clientproxy.xml
/etc/xroad/jetty/ocsp-responder.xml
/etc/xroad/services/
/etc/xroad/services/local.properties
/etc/xroad/devices.ini
/etc/xroad/db.properties
/etc/xroad/signer/
/etc/xroad/signer/softtoken/
/etc/xroad/signer/softtoken/C0DEF9CE56F247DC12711470915AAC499CD1C550.p12
/etc/xroad/signer/softtoken/252D1004A9AE0EB1B500366C67AAF90FC19A517F.p12
/etc/xroad/signer/softtoken/B4446F29352ED3367D8AC5D5D86FA6BBE4308EEC.p12
/etc/xroad/signer/softtoken/.softtoken.p12
/etc/xroad/signer/keyconf.xml
/etc/xroad/conf.d/
/etc/xroad/conf.d/signer.ini
/etc/xroad/conf.d/proxy.ini
/etc/xroad/conf.d/proxy-ui-api.ini
/etc/xroad/conf.d/signer-logback.xml
/etc/xroad/conf.d/signer-console-logback.xml
/etc/xroad/conf.d/common.ini
/etc/xroad/conf.d/proxy-ui-api-logback.xml
/etc/xroad/conf.d/addons/
/etc/xroad/conf.d/addons/messagelog-archiver-logback.xml
/etc/xroad/conf.d/addons/monitor-logback.xml
/etc/xroad/conf.d/addons/monitor.ini
/etc/xroad/conf.d/addons/message-log.ini
/etc/xroad/conf.d/confclient-logback.xml
/etc/xroad/conf.d/confclient-logback-service.xml
/etc/xroad/conf.d/local.ini
/etc/xroad/conf.d/proxy-logback.xml
/var/lib/xroad/dbdump.dat
Backup file saved to /var/lib/xroad/backup/mybackup.tar

=== Warning! ===========================================================================
The file “/etc/xroad/services/local.conf” used for configuration overrides is deprecated
and not included in the backups anymore.
The file “/etc/xroad/services/local.properties” should be used instead.
=== Warning! ===========================================================================

```